### DISCUSSION TITLE
Just some feedbacks about documentations for beginners

### DISCUSSION THREAD

**Discussion by thangngoc89** [vikejs/vike#1646] [Discussion]
Hey, 

Thank you for making vike. I was search for a quick way to generate a static with MDX and react using vite (SSG) .
For context, I already has an app built using react + react router + vite and that part is always behind a login so just pure SPA.

vite-plugin-ssr was the first hit and it leads me here.

## TL;DR

- Add obvious getting started link in the home page
- Copy the simplest possible Bati command to the Scafford new app page.

## Long version

So this is the flow of me, completely new to vike trying to getting started:

- https://vike.dev doesn't contain any actions to actually getting started.
- Scafford new app page. So after looking through the sidebar, I could see Getting Started > Scafford new app page.
<img width="891" alt="image" src="https://github.com/vikejs/vike/assets/3049054/3c79d853-6bc0-4b1f-ac1d-cdba507e3731">

Naturally I would skip the Bati part because that's another unfamiliar name, maybe some advertisement, etc, ...

Next part with runnable command has a big warning. 

- From there I went back to home page hoping to see another Getting started section but there are none (or at least not obvious that I've missed it somehow)
- So I decided to went back to Scafford new app page and clicked on the Bati link.
- It seems like a good reference but it uses `pnpm`. I've heard many good things about `pnpm` but I don't have it on my system. 
- So I went back to Scafford new app page and try the **expert** section instead.

**Comment by brillout** [comment]
Thank you for you feedback.

One thing that I think is very much lacking is a migration guide to add SSR/SSG to an existing app. Which seems to be your use case? I went ahead and added a comprehensive migration guide:
 - https://github.com/brillout/vite-to-vike
 - [`docs: add Vite -> Vike migration guide`](https://github.com/vikejs/vike/commit/7b676ca7d97737a68051b00114723c382c46c123)

Also: [`docs: improve /new`](https://github.com/vikejs/vike/commit/a3eaa67ec1e9e2d01f2bf19e5025c19a0eeeecbd)

I think this covers the issues you had?

As for your Bati feedback, CC @magne4000.

**Reply by thangngoc89** [reply]
I think the links you listed pretty much covered most of the issues I had. What I ended up doing was created a new project with Bati on StackBlitz and copy the necessary bits to my project.

**Reply by brillout** [reply]
I further added some polish to https://vike.dev/add and https://vike.dev/new, hopefully it's clearer now.

**Reply by thangngoc89** [reply]
@brillout I would recommend visible links to `/add` and `/new` from https://vike.dev/

**Reply by brillout** [reply]
Agreed and that's the plan with the upcoming brand-new landing page. Also a guided tour showcasing what makes Vike special.

**Comment by magne4000** [comment]
> Naturally I would skip the Bati part because that's another unfamiliar name, maybe some advertisement, etc, ...

I think that adding better context around Bati is a good idea, exactly because of this point.
I'm not sure if adding some default command is the right idea, let me think about that. A screenshot or PNGA/GIF of Bati that redirects to Bati could be an alternative solution.

**Reply by thangngoc89** [reply]
@magne4000 Probably what put me off more is Bati requires pnpm.

**Reply by magne4000** [reply]
In fact it actually doesn't, it's mostly for simplifying the UI, but it works with all package managers. But that's a good point, I understand that it can throw people off. I'll think of a solution.

**Reply by magne4000** [reply]
> Probably what put me off more is Bati requires pnpm.

Bati's website now shows support for pnpm, yarn, bun and npm.
